prisma/migrations/20250514145741_unique_device_identity_on_user_id_and_xmtp_id/migration.sql

```diff
@@ -0,0 +1,35 @@
+/* Query to find DeviceIdentity records that should be deleted */
```

Comment:
Not sure how Prisma migrations parse, but you might need to name this statement to make Prisma run it. They have names on the auto-generated statement below.

`-- DeleteDuplicates`

Reply:
Haven't found anything about that? 😮 From what I found it should run the sql correctly